### ISSUE TITLE
CCU bleibt im Startbildschirm hängen/Betrieb nur mit HM-LGW

### ISSUE THREAD

**Issue by MiniAdnim** [leonsio/YAHM#181]
Hallo Leonsio,
laut CCU2 Log ist "no BID-COS Interfaces available" und die CCU startet nicht durch. 

Ist ein Betrieb nur mit LGWs möglich oder geht das nicht? 
und muss ich dazu spezielle Einstellungen vornehmen?wenn ja steht irgendwo was genau?

Danke Dir vielmals!

**Comment by leonsio** [comment]
Hi

Ja ist möglich. Läuft bei mir aktuell so
Ggf kann Startvorgang lange dauern. Die gui sollte irgendwann aber kommen

Von meinem iPhone gesendet

**Comment by pfink** [comment]
Ja, bei mir dauert der (erstmalige) CCU-Start auch bis zu 15 Minuten, obwohl keinerlei Last erkennbar ist. Habe auch gerade 2 Stunden nach einem nicht vorhandenen Problem gesucht ;-)
